app/src/lib/parser/pairings.test.js

```diff
@@ -47,9 +47,18 @@ function create_lookup_result(stem, {sense='A', part_of_speech='Noun', level=1}=
 	}
 }
 
+/**
+ * 
+ * @param {Token[]} tokens 
+ * @returns {Sentence}
+ */
+function create_sentence(tokens) {
+	return { clause: create_clause_token(tokens) }
+}
+
 describe('pairing level check', () => {
 	test('both words right level', () => {
-		const test_tokens = [
+		const test_tokens = [create_sentence([
 			create_pairing_token(
 				create_lookup_token('first', {lookup_results: [create_lookup_result('first', {level: 0})]}),
```

Comment:
At some point it might be good to rework the tests to use a mock function for lookups. I could add an issue for that?